app/src/main/java/com/kickstarter/viewmodels/AccountViewModel.kt

```diff
@@ -94,10 +105,22 @@ interface AccountViewModel {
             return this.progressBarIsVisible
         }
 
+        override fun showEmailErrorIcon(): Observable<Boolean> = this.showEmailErrorIcon
+
         override fun success(): BehaviorSubject<String> {
             return this.success
         }
 
+        private fun showEmailErrorImage(userPrivacy: UserPrivacyQuery.Data?): Boolean? {
+            return if (!userPrivacy?.me()?.isDeliverable!!) {
```

Comment:
I got some helpful feedback from @Scollaco. When you use `!!` and this value is null, it's going to crash the app. We shouldn't assume that it's not going to be null. Here's a helpful link I found: https://kotlinlang.org/docs/reference/null-safety.html

Reply:
Fixed!

Reply:
amazing